### DISCUSSION TITLE
AdjustContrast has  notype of “invert_image”

### DISCUSSION THREAD

**Discussion by Wangrui-berry** [Project-MONAI/MONAI#6734] [Q&A] [answered]
AdjustContrastd(keys=("t2","ap","pvp"), gamma=0.5,invert_image=False,retain_status=False),

TypeError: __init__() got an unexpected keyword argument 'invert_image

**Comment by KumoLiu** [comment, marked as answer]
Hi @Sudayberry, can't reproduce the error.
```
from monai.transforms import AdjustContrastd

x = torch.rand([5, 1, 64, 64, 64])
data = {"t2": x}

out = AdjustContrastd(keys=("t2"), gamma=0.5,invert_image=False, retain_stats=False)(data)
print(out["t2"].shape)
```

BTW, it's `retain_stats` instead of `retain_status`.

Convert to discussion for now, feel free to create another issue. Thanks!
